WUM/wum-sce-test-wso2ei-6.1.0-full-testgrid.yaml

```diff
@@ -18,6 +19,22 @@ infrastructureConfig:
       remoteBranch: "6.1.0"
       description: Provision Infra for running scenario tests
       scripts:
+        - name: 'prod-wso2-ei-scenario-vfs-be-deployment'
+          description: ''
+          type: CLOUDFORMATION
+          file: integrator/Sample-Backends/vfs-be.yaml
+          inputParameters:
+            KeyPairName: "deployment-prod"
+            AMI: "ami-01b72d1b13ca0b4bc"
+            FTPUserName: "wso2user"
+            FTPUserPassword: "FTPUserPassword"
+        - name: 'prod-wso2-ei-scenario-activemq-be-deployment'
+          description: ''
+          type: CLOUDFORMATION
+          file: integrator/Sample-Backends/activemq-be.yaml
+          inputParameters:
+            KeyPairName: "deployment-prod"
+            AMI: "ami-01b72d1b13ca0b4bc"
```

Comment:
Shall we move AMIs into the cloudformation itself? You need to use AMIs that is available in the aws region where this is getting executed. Now that we are scaling to use multiple aws regions, we need a more scalable model.

Specifically, we are going to use following regions - 
us-east-1
us-east-2
us-west-1
us-west-2
eu-west-1
ap-southeast-2

Like this - https://github.com/wso2-support/aws-ei/blob/6.1.0/integrator/Minimum-HA/integrator-ha.yaml#L1239-L1257

Reply:
Added. Please refer following pull requests.

https://github.com/wso2-support/aws-ei/pull/100
https://github.com/wso2-support/aws-ei/pull/102
https://github.com/wso2-support/aws-ei/pull/103
https://github.com/wso2-support/aws-ei/pull/104
https://github.com/wso2-support/aws-ei/pull/105
https://github.com/wso2-support/aws-esb/pull/33
https://github.com/wso2-support/aws-esb/pull/34